pkg/autoscaler/autoscaler.go

```diff
@@ -93,6 +93,11 @@ func (a *Autoscaler) Start() {
 		}
 
 		for deploymentName, waitCount := range stats.ActiveRequests {
+			// TODO remove this check and ensure only stats for deployments with models are returned
+			if !a.Deployments.HasModel(deploymentName) {
```

Comment:
What scenario leads lingo to fall into this state?

Reply:
Lingo right now has autoscaler active on all deployments because ActiceRequests returns the active requests for all deployments. This if statement can be removed once we fix https://github.com/substratusai/lingo/issues/59 and use Pod endpoints and only add endpoints of Deployments that have a lingo model annotation.

Reply:
To answer your question, this happens in all scenarios.